simulation: True
visualization: True
FOV: 1.57
min_footprint: 2
speed: 155
policy: semi_greedy
area_width: 200.0
area_height: 200.0
area_rotation: 0.0
logging: false
log_folder: /home/abbas/ros_workspace/src/active_survey/log
patches: 20
percent_interesting: 10.0
auto_exit: false
bypass_controller: false
prior_file_name: priors/prior10.jpg
time_limit: 4000
turning_time: 10.0
random_seed: -1
gp_sigma: 0.1
exploitation_rate: 0.005
coarse_coverage_height: 8.0
sensing_height: 1.0
average_speed: 1.0
uncertain_cells_threshold: 1.0
